### PR TITLE
USSD registration dynamic text functionality for voice end state

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr please review

File: src/ussd_registration.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ go.app = function() {
             "state_voice_times":
                 "Thank you. At what time would they like to receive these calls?",
             "state_end_voice":
-                "Thank you. The person will now start receiving calls on [day and day] between [time - time].",
+                "Thank you. The person will now start receiving calls on [{{first_day}} and {{second_day}}] between [{{start_time}} - {{end_time}}].",
```
Comment (gsvr) [inline src/ussd_registration.js]: Drop the square brackets

File: src/ussd_registration.js [hunk 2]
```diff
@@ -333,8 +333,46 @@ go.app = function() {
...
+            var first_voice_day;
+            var second_voice_day;
+            var voice_start_time;
+            var voice_end_time;
+
+            if (self.im.user.answers.state_voice_days == 'mon_wed') {
+                first_voice_day = 'Monday';
+                second_voice_day = 'Wednesday';
+
+                switch (self.im.user.answers.state_voice_times) {
```
Comment (gsvr) [inline src/ussd_registration.js]: Move the switch outside the if block then you only have to do it once.

Comment (NicolaasDieBaas) [inline src/ussd_registration.js]: of course duh! 
Thanks!

**Comment by NicolaasDieBaas** [comment]
@gsvr 2nd pass review please

**Comment by gsvr** [comment]
:+1:

**Pull request merged**